spec/classes/init_spec.rb

```diff
@@ -654,6 +654,68 @@
       }
     end
 
+    describe 'with ruby 2.2 with switch' do
```

Comment:
Technically you won't need to spec test against ruby 2.2 or 2.3 on Puppet 4 yet because it still uses 2.1 internally I believe.